lib/index.js

```diff
@@ -15,7 +15,8 @@ const brPermissionCheck = promisify(brPermission.checkPermission);
 const database = require('bedrock-mongodb');
 const jsonpatch = require('fast-json-patch');
 const validateCapabilityDelegation = promisify(
-  brIdentity.validateCapabilityDelegation);
+  brIdentity.validateCapabilityDelegation
+);
```

Comment:
lint rules shouldn't make us do this

Reply:
but the opposite is ok?

Reply:
the real issue is really with callbacks.
when we have callbacks in paramaters they take up multiple lines which results in
odd scenarios. anyways I think they we can get the behavior you want. did not think of this case
line break then no line break when setting up the rule this morning.

Reply:
Yes, this should be an error:

```js
promsify(params
) // errors
```

Reply:
> the real issue is really with callbacks.

... and we have a goal of removing callbacks everywhere (eventually) so that should go away or at least be largely mitigated.